cirq/sim/wave_function.py

```diff
@@ -570,14 +570,16 @@ def _probs(state: np.ndarray, indices: List[int],
 
     # Calculate the probabilities for measuring the particular results.
     meas_shape = tuple(qid_shape[i] for i in indices)
-    probs = [
-        np.linalg.norm(tensor[linalg.slice_for_qubits_equal_to(
-            indices, big_endian_qureg_value=b, qid_shape=qid_shape)])**2
+    probs = np.abs([
+        tensor[linalg.slice_for_qubits_equal_to(indices,
+                                                big_endian_qureg_value=b,
+                                                qid_shape=qid_shape)]
         for b in range(np.prod(meas_shape, dtype=int))
-    ]
+    ])**2
```

Comment:
FYI (no action needed): I imagine we're still computing square roots behind the scenes only to square them later. We can revisit this for additional speedup after https://github.com/numpy/numpy/issues/13179.